### PR TITLE
adding index.html and style.css file

### PR DESCRIPTION
**Pull request by okusarobert**
This PR consists of replicating the The Next Web website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Great job on your project. However, there are some required changes before the project can be approved.**

## Mandatory

### Large screen (min-width: 1024 px)
**Note: min-width is 1024px for large screens and not 1440px as used.**

1).  **The navigation bar and the top section of your page does not look like the original and here is why:**
  

original | clone
---------| -------
navtnw_original | navtnw_clone

- **The TNW logo should not be displayed on the top leftmost part but should be a small logo on the rightmost part next to the clock and search icon.**

- **Topmost menu,(with a black background), should be displayed on the leftmost part of the page.**

- **The small text in the yellow background overlaps bottom section with background images.**

- **For the sections that have a background image, some of the text and icons are hidden. Also, reduce the text font-size.**

2). **The topmost menu, (with a back background ), should not have a sticky position.**

3). **Latest News section**

original | clone
---------| -------
latest_original | latest_clone

- **Each row should display four images instead of three.**

- **There is some text displaying behind the images. (Check the first and second images below the title).**

- **There is some text next to the Latest news title that should not be there.**

- **Reduce the headline font-size.**

4). **Latest Funding round Section**

original | clone
---------| -------
funding_original | funding_clone

- **All the titles are displayed on one line.**

- **The row is divided into four sections of content and not three.**

5). **The sections after.**

original | clone
---------| -------
hardfork_original | hardfork_clone

- **Each row displays three images instead of two.**

- **The rows content overlap each other.**

- **Add a list-style-type to the lists.**

- **Reduce the content font-size so it doesn't look as big as the image.**

6). **For the latest deals section, the row displays four images and not three.**

### Medium screen ( min-width: 768px)

1). **The TNW logo should not display on the medium screen.**

original | clone
---------| -------
tnwmedium_original | tnwmedium_clone



### Small screen

1).  **Navigation bar and the top section of your page.**

original | clone
---------| -------
smalltnw_original | smalltnw_clone

- **Please fix the icon display so it looks like the original.**

- **I can see page content, above the yellow background, as I scroll down the page.**

- **The sections with image backgrounds either don't have text content or some parts of the contents are hidden.**

2). **For the other sections of the page, reduce the text font-size especially for the content with background images.**

3). **Increase the background image height for the other sections as well. See the example below.**

original | clone
---------| -------
hfsmall_original | hfsmall_clone

- **Also fix the font-size for the lists. (Check the screenshot above).**

#### Happy coding 😃.
**PS: You can reach me on slack(Rose Wanjohi).**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1771 @@
...
+    justify-content: space-between;
+  }
+
+  .mt-5 {
+    margin-top: 50px;
+  }
+
+  .fx-navbar .nav-img {
+    display: block;
+    position: static !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1771 @@
...
+    flex-direction: column;
+  }
+
+  .under-nav-msg {
+    margin-top: 0%;
+    position: static !important;
+    justify-content: flex-start;
+    order: 3;
+    gap: 0;
+    z-index: auto !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1771 @@
...
+    white-space: nowrap;
+  }
+
+  .left-side-links ul {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .navbar2 {
+    display: none !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1771 @@
...
+    margin-top: 100px;
+  }
+
+  .navbar2 {
+    height: 43px;
+    justify-content: space-between;
+    align-items: center;
+    top: 0;
+    z-index: auto !important;
+    position: static !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1771 @@
...
+    flex: 3;
+  }
+
+  .under-nav-msg {
+    top: 10.2%;
+    justify-content: flex-start;
+    font-size: 0.9vw;
+    position: static;
+    z-index: auto !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1771 @@
...
+  }
+
+  .fx-navbar2 {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    align-items: center;
+    top: 4.9%;
+    background-color: white;
+    position: static;
+    z-index: auto !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove `!important`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Takhmina175 and @okusarobert
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Fix spacing issues as @blackpintz explained in her review
- [ ] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)

### Header
- [ ] Manage the header text in your clone

|[Source Page](https://thenextweb.com/)|[Clone Page](https://takhmina175.github.io/theNextWeb_ClonePage/)|
|:---:|:---:|
|source-page|clone|

### Latest Funding
- [ ] manage the links on 813px

|[Source Page](https://thenextweb.com/)|[Clone Page](https://takhmina175.github.io/theNextWeb_ClonePage/)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :construction:

### Hi team :wave: 
Good job with the changes! :ok_hand: We are almost there.
However, we do have some changes to carry out before we proceed (see comments below).

**[ REQUIRED CHANGES ]** 

- [ ] The main navigation section and the sub navigation section should be rendered at large screen sizes.
- Right now, you're only rendering the sub-navigation section for large screen sizes (1024px and above).

Source page:


Clone page:


- [ ] Remove the `TNW` logo on top of the featured article for medium screen sizes (768px to 1023px)

Source page:


Clone page:


- [ ] You have some issues to fix in the footer section.



- On large screen sizes, the footer occupies a single line. **Note that it does not change across breakpoints.** This implies that if the footer breaks, it should be driven by the content.

Source page:


Clone page:


- On medium small screen size, the social media icons seems to be overlapping.

| Source page      | Clone page |
| ----------- | ----------- |
|       |        |


* **When you have made the specified changes, please submit another code review. Thank you!**
* **If you feel like you need help or you have any question(s), I'm @taofeekolalere on slack**

Regards,
[Taofeek Olalere](https://github.com/teekaytech)
Happy Coding :technologist:

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello @Takhmina175 and @okusarobert

Your progress is amazing, well done on implementing all the requested changes 👍 

There are still some changes that are necessary for this project to be approved.

- [ ] Looks like you lost the **TNW** on the larger screen after fixing it for the medium screen. See screenshot below

- [ ] [Optional] There is a gradient overlay that allows the text to be more pronounced.
feature

- [ ] Your footer on the large screen still has some work to do, see the screenshot below. 
- The spacing is too much compared to the original
- The spacing between the social media icons is much
- Increase the size of the menu items below the social media icons
- Increase the social media icon size and reduce its padding
Just compare the clone and the original, it doesn't have to be perfect, close enough is good.
Perfect would be awesome though 😄 
tnwfooter

Fix these changes and submit another code review request and you should be good to go.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
What amazing work you have been doing in this project! 👍 
Just a few more steps to get it approved 💪 

Kindly, check my comments below 👇  apply the changes and request a new code review ⏩ 

Good luck! 😃 

## Changes
- You have an overlay for screens below 1024px, kindly fix it.
Capture
- The main navbar at 1024px does not match the original, please modify it
original:
Capture2
Your project:
Capture1
- For small screens, 768px and below, the text is missing in the main section
Capture3
- For small devices, the hard-fork section has a problem with the list, it should match the width of the photo and the rest of the item
Your project:
Capture4
Original:
Capture5
- In this same section, the space between the list items must be bigger and the image size also must be bigger, kindly fix it.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done well implementing the changes required by the previous reviewers. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] As the previous reviewer mentioned, try to add some space after each section. You can also reduce the font size and boldness so it matches the source page. Right now, things look so clumsy in your page.



Source page:


Clone page:


- [ ] As an improvement to what the immediate past reviewer mentioned, you should improve on the general layout of your `Insights`, `Podium`, `Creative`, `Tech` and similar sections of your page

- As an example, see the `Creative` section below and observe the difference, then use it to modify others.

Source page:
 

Clone page: 
 |

**[ OPTIONAL CHANGES ]**

- [ ] Please pay attention to margins and paddings as seen in the source page.

**Please do not give up on these changes, the project is close to completion. Kindly make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Good job implementing some changes 👍 
But we still need some changes to do before approving this project 💪 
Kindly, check my  comments below 👇  and after they are done resubmit your project for a new code review

## Changes
- Latest findings section must have only 3 items in screens below 1024px, please remove the extra item for mobile screens
- In screens above 1024px all your sections have an extra horizontal margin, please make it look like the original.
- For small screens, 768px and below, the text is still missing in the main section
- As the previous TSE mentioned, please fix the image's width in the latest section in the Insights, Podium, Creative, Tech sections. You can remove the horizontal padding and add display flex to the parent div, then justify-content as required.
- In the Insights, Podium, Creative, Tech sections for screens above 1024px there is too much margin-bottom, while in smaller screens I can still see overlay(between 1024px and 810px), kindly fix it.

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @Takhmina175 !!

You are almost done :) 🌟 great job implementing almost all required changes

Kidly check below the 3 remaining issues (extra space, horizontal scrollbar and overlap):

- **[REQUIRED]**  Please check the differences below:
**TABLET**
- [ ] Please see the annotations and fix them:
image
- [ ] You still have overlayed content between sections, to fix it, just remove the fixed height and let it accommodate by itself:
```
@media (min-width: 768px)
.container-height {
    height: 670px;                                 <------- delete this
}
```
image

**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

- **OPTIONAL** Not required since it was not required by previous TSE, but for next time, pay attention to details like these:
image
image

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao
##### 🌟  If you have feedback about my reviews I would appreciate if can leave a recommendation in LinkedIn 💌 or let me know in Slack if you see improvements to be done 😛 

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Takhmina175 and @okusarobert
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Remove commented code from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] Fix the navbar issue as mentioned before
### Funding section
- Fix the text overlapping issue on medium and small screens

|[Source Page](https://thenextweb.com/)|[Clone Page](https://takhmina175.github.io/theNextWeb_ClonePage/)|
|:---:|:---:|
|source-page|clone|

|[Source Page](https://thenextweb.com/)|[Clone Page](https://takhmina175.github.io/theNextWeb_ClonePage/)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — COMMENTED** [review]
# Congratslutons  Approved :tada:
-------------------------------------------------------------------------------------------------
Hello Developer :wave:
You are on :fire:
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. :clap:
Please keep working hard and move forward with your curriculum. 
Looking forward to seeing your next project. :smile:

**To highlight** 

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
**Optional**  Please make your PR more descriptive by adding more details about your project. This can help TSEs to review your project and your PR looks professional.  
**optional** Please add node_modules in the gitignore file because every system having node install on their system can easily get without any problem. The main reason is that node_modules usually cover a lot of space and cloning your project becomes hard. 
--------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
@arslan Bisharat on slack in the case you want to discuss anything.**

**Pull request merged**